.gitignore

```diff
@@ -1,4 +1,5 @@
 *.env
+.idea/
```

Comment:
@Kludex I'm guessing you don't want this?

Reply:
Yep, I don't agree with adding environment specific to the `.gitignore`. Users can and should add `.idea` to the global `.gitignore`.

Reply:
```suggestion
```